narwhals/typing.py

```diff
@@ -29,6 +29,10 @@ def columns(self) -> Any: ...
 
         def join(self, *args: Any, **kwargs: Any) -> Any: ...
 
+    class NativeSeries(Protocol):
+        @property
+        def name(self) -> Any: ...
```

Comment:
not sure this is true in general unfortunately 😭 
```python
In [4]: pa.chunked_array([['a', 'b', 'c']]).name
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[4], line 1
----> 1 pa.chunked_array([['a', 'b', 'c']]).name

AttributeError: 'pyarrow.lib.ChunkedArray' object has no attribute 'name'

```

let's just put `__len__`?

Reply:
Thank you for the correction. I didn't consider the situation.